.github/workflows/tests.yaml

```diff
@@ -62,7 +62,14 @@ jobs:
       run: tuist install --path SampleApp
 
     - name: Build & Test
-      run: tuist test --path SampleApp --os ${{ matrix.sdk }} UnitTests
+      run: tuist test --path SampleApp --os ${{ matrix.sdk }} UnitTests -- -resultBundlePath TestResult
```

Comment:
This was used for debugging but might be worth keeping around, is there a reason we dont save the test results presently?